### PR TITLE
Privacy policy updates June 2023

### PR DESCRIPTION
**Pull request by dmcavoy**
Updates the privacy policy to match requests from Travis. Made a copy of the old version to keep the past record.

Last update for reference of the process: https://github.com/code-dot-org/code-dot-org/pull/49523

### CODE REVIEW

File: dashboard/app/views/home/_privacy.md [hunk 1]
```diff
@@ -58,7 +58,9 @@ Code.org is a signatory to the [Student Privacy Pledge](https://studentprivacypl
...
+We generally collect personal data in three ways: (1) information a User voluntarily provides to us by using the Services, (2) information we automatically collect as a User uses the Services, and (3) information from third-parties. The types and amounts of personal data collected vary depending on whether the User is a Student, Teacher, parent or other visitor, and on how they use the Services, but can be generally categorized as (a) account registration data (such as username, password, email address), (b) demographic data (such as age, gender, race), (c)   .technical data (such as browser or device, IP address; login times), (d) User uploaded data (such as images and sounds), (e) User response or feedback data (such as survey responses and notes to and from Teachers); (f) platform usage data (such as progress data, projects created), and (g) contact data for non-curriculum processes (such as email addresses for educators, parents, and others who wish to receive newsletters and other updates).
```
Comment (megcrenshaw) [inline dashboard/app/views/home/_privacy.md]: Not: third parties should be two separate words here

Comment (megcrenshaw) [reply dashboard/app/views/home/_privacy.md]: The formatting in part c) is off: there's extra spacing, a period, and I'm not sure why there's a semicolon in the parens (it looks like a normal list to me)

File: dashboard/app/views/home/_privacy.md [hunk 2]
```diff
@@ -148,7 +150,7 @@ In select courses, such as the Code.org CS A AP course, Teachers may optionally
 
 In addition, Teachers may provide written feedback to their Students on coursework. Users may also send messages to Code.org for customer-support requests.
 
-Other than the above, the Services do not directly provide any other form of messaging among Users (although Users may use Code.org tools to create apps that support messaging among individuals).
+Other than the above, the Services do not directly provide any other form of messaging among Users (although Users could use Code.org tools to create apps that support messaging among individuals).
```
Comment (megcrenshaw) [inline dashboard/app/views/home/_privacy.md:153]: I thought we actively discouraged chat creation for safety reasons? Is that not correct?

File: dashboard/app/views/home/_privacy.md [hunk 3]
```diff
@@ -253,17 +265,16 @@ Deleting your Code.org account will not delete data that you may have submitted
...
+Code.org may be used in a classroom setting by schools, school districts, or teachers (collectively referred to as “Schools”). When the Services are used as part of a School’s educational curriculum, the personal data related to the School’s Student Users that is provided to Code.org by the School or collected by Code.org during the provision of the Services to a School, may include information defined as “educational records” by the Family Educational Rights and Privacy Act (“FERPA”) or be covered by other similar student data privacy laws, to which the School may be subject. In these cases, the Student’s personal data we collect, along with any other student records the school provides to us and any student-generated content is referred to as “Student Records” and we have implemented controls and procedures to help the Schools address their obligations under such laws. In some cases where we process Student Records, we may enter into separate agreements with Schools that supplement our Terms of Service, such as our [Student Data Privacy Addendum. ](/dpa)
```
Comment (megcrenshaw) [inline dashboard/app/views/home/_privacy.md]: Nit: extra space at the end before the closing link bracket

File: dashboard/app/views/home/_privacy.md [hunk 4]
```diff
@@ -253,17 +265,16 @@ Deleting your Code.org account will not delete data that you may have submitted
...
 
 <a name="protecting-childrens-privacy"></a>
```
Comment (megcrenshaw) [inline dashboard/app/views/home/_privacy.md]: Should this link change for consistency?

File: dashboard/app/views/home/_privacy.md [hunk 5]
```diff
@@ -274,6 +285,46 @@ The Code.org Services are designed for Users of all ages, including children und
...
+
+#### How Parents can provide consent (where required)
+
+Although parental consent is not generally required, in those jurisdictions where we require such consent in connection with a Child account the parent may provide consent by responding affirmatively to an email sent by Code.org to the Parent’s email address provided by the Child during registration.  We will send a reminder email after a few days if no response is received, but if we do not receive consent within the time prescribed, the Child’s account will be closed and all account information (including the parent’s email information) will be deleted from our systems. If parental consent is received, the Parent’s email address will be automatically linked to their child’s account to stay updated on their Child’s progress and projects - and the Parent email address can also be used for password recovery and to request support (including account deletion at any time) for the Child’s account. Parents will need to ensure they know their child’s email address and username for support requests.
```
Comment (megcrenshaw) [inline dashboard/app/views/home/_privacy.md]: Nit: comma at "in those jurisdictions ... with a child account**,** the ..."

**Review by megcrenshaw — APPROVED** [review]
I only found some small nits, and I'm not sure they're even necessary to update.

I did not carefully review the old privacy policy, nor the files in the "public" folder (which coincide with files I did review).

File: dashboard/app/views/home/_privacy.md (hunk 2 above)
Comment (dmcavoy) [reply dashboard/app/views/home/_privacy.md:153]: We do. I think this is a covering our bases thing.

**Comment by dmcavoy** [comment]
Thanks for the review Meg! I'm going to go ahead and merge ahead of drone to give Travis time to review on staging before the deploy

File: dashboard/app/views/home/_privacy.md (hunk 2 above)
Comment (megcrenshaw) [reply dashboard/app/views/home/_privacy.md:153]: Yep, that makes sense

**Comment by megcrenshaw** [comment]
Looks good, Dani! When I approve, I don't feel like I need to take another look, so all good there 👍

**Pull request merged**